chef/cookbooks/hyperv/recipes/openstack_install.rb

```diff
@@ -41,7 +41,7 @@
 
 windows_batch "unzip_neutron" do
   code <<-EOH
-  #{node[:sevenzip][:command]} x #{node[:cache_location]}#{node[:openstack][:neutron][:file]} -o#{node[:openstack][:location]}\\dist -r -y
+  #{node[:sevenzip][:command]} x #{node[:cache_location]}#{node[:openstack][:neutron][:file]} -o#{node[:openstack][:location]} -r -y
   #{node[:sevenzip][:command]} x #{node[:openstack][:location]}\\dist\\#{node[:openstack][:neutron][:name]}-#{node[:openstack][:neutron][:version]}.tar -o#{node[:openstack][:location]} -r -y
   rmdir /S /Q #{node[:openstack][:location]}\\dist
```

Comment:
Hrm, forgot to press "comment", I think: @ovisan I'm not sure I understand how removing sdist from the first line and not from the other lines works. What was the original error?